tests/test_distributions.py

```diff
@@ -3,15 +3,20 @@
 
 from stable_baselines3 import A2C, PPO
 from stable_baselines3.common.distributions import (DiagGaussianDistribution, TanhBijector,
-                                                   StateDependentNoiseDistribution,
-                                                   CategoricalDistribution, SquashedDiagGaussianDistribution)
+                                                   SquashedDiagGaussianDistribution,
+                                                   CategoricalDistribution,
+                                                   MultiCategoricalDistribution,
+                                                   BernoulliDistribution,
+                                                   StateDependentNoiseDistribution)
 from stable_baselines3.common.utils import set_random_seed
 
 
 N_ACTIONS = 2
 N_FEATURES = 3
 N_SAMPLES = int(5e6)
 
+N_ACTIONS_MULTI = [4,3,2]
```

Comment:
minor: please reformat to comply with PEP8 (you can use PyCharm for doing that automatically)